### ISSUE TITLE
Cleanup Testrail data - firefox-ios

### ISSUE THREAD

**Issue by rpappalax** [mozilla-mobile/mobile-test-eng#543]
Waiting on Smoke Test suite to be merged into Full Functional

**Comment by isabelrios** [comment]
Smoketest is done. Still work needed for Full Functional test suite

**Comment by isabelrios** [comment]
Starting to work on Full Functional test suite

**Comment by isabelrios** [comment]
I think we can close this for now. Tests have been updated in Test Rail with the Automation Coverage. There are TODOs set for updating manual tests/update automated tests/add missing steps and so but that will require more concrete tasks
